includes/class-civicrm-caldera-forms-helper.php

```diff
@@ -484,13 +484,43 @@ public function map_fields_to_prerender( $config, &$form, $ignore_fields, $entit
 				if ( $field['type'] == 'radio' ) {
 					$options = Caldera_Forms_Field_Util::find_option_values( $field );
 					$form['fields'][$field['ID']]['config']['default'] = array_search( $value, $options );
+				} elseif ( $field['type'] == 'date_picker' ) {
```

Comment:
I think this should be in it's own statement, and placed **after** the `$value` declaration on line 469 (`$value` holds the Civi data) and **before** the filter where the _defaults_ (Civi data) are assigned on line 482.

Reply:
OK, no problem.